README.md

```diff
@@ -2,52 +2,43 @@
+* [Checklist dataset on GBIF](https://doi.org/10.15468/yxcq07)
 
-### Checklist dataset
+---
 
-* Description of the [Darwin Core mapping](src/dwc_checklist.md) (= a rendition of the [mapping script](src/dwc_checklist.R))
-* Generated [Darwin Core files](data/processed/dwc_checklist/)
-* Published [dataset on the IPT](https://ipt.inbo.be/resource?r=alien-macroinvertebrates-checklist)
-* Published [dataset on GBIF](https://doi.org/10.15468/yxcq07)
+* [Occurrence dataset on the IPT](https://ipt.inbo.be/resource?r=alien-macroinvertebrate-occurrences)
+* [Occurrence dataset on GBIF](https://doi.org/10.15468/xjtfoo)
+* [Boets et al. 2016](http://www.aquaticinvasions.net/2016/AI_2016_Boets_etal.pdf): Data paper describing the occurrence dataset and methodology
 
 ## Repo structure
 
-The repository structure is based on [Cookiecutter Data Science](http://drivendata.github.io/cookiecutter-data-science/). Files indicated with `GENERATED` should not be edited manually.
+The repository structure is based on [Cookiecutter Data Science](http://drivendata.github.io/cookiecutter-data-science/). Files and directories indicated with `GENERATED` should not be edited manually.
 
 ```
-├── README.md               : Top-level description of the project and how to run it
-├── LICENSE                 : Project license
-├── .gitignore              : Files and folders to be ignored by git
+├── README.md         : Description of this repository
+├── LICENSE           : Repository license
+├── .gitignore        : Files and directories to be ignored by git
 │
 ├── data
-│   ├── raw                 : Source occurrence data, not maintained elsewhere,
-|   |                         input for both mapping scripts and CARTO 
-│   └── processed
-|       ├── dwc_occurrence  : Darwin Core output of occurrence mapping script GENERATED
-|       └── dwc_checklist   : Darwin Core output of checklist mapping script GENERATED
+│   ├── raw           : Source data, input for mapping script
+│   └── processed     : Darwin Core output of mapping script GENERATED
 │
-├── src
-│   ├── dwc_occurrence.R    : Darwin Core mapping script for occurrence dataset
-│   ├── dwc_occurrence.md   : Nicer rendition of above script, created by knitr::spin GENERATED
-│   ├── dwc_checklist.R     : Darwin Core mapping script for checklist dataset
-│   ├── dwc_checklist.md    : Nicer rendition of above script, created by knitr::spin GENERATED
-│   └── src.Rproj           : RStudio project file
+├── docs              : Repository website GENERATED
 │
-└── docs                    : Code for a one-page website to explore occurrence data on CARTO
-                              http://trias-project.github.io/alien-macroinvertebrates
+└── src
+    ├── dwc_checklist.Rmd  : Darwin Core mapping script for checklist dataset
+    ├── dwc_occurrence.Rmd : Darwin Core mapping script for occurrence dataset
+    └── src.Rproj          : RStudio project file
 ```
 
 ## Installation
```

Comment:
This installation guide is still based on our previous approach to use knitr and is outdated. I will correct this when I update the manual of alien plants (planned coming Friday). However, I think we should repeat the full procedure here. I think it's nicer to provide all information in the README, rather than referring to another repo.

Reply:
Great! I referenced because I didn't want to necessarily repeat the info... but it shouldn't change really once settled.